vignettes/howtogenefinder.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "How to find genes whose expression profile is similar to that of
+specified genes"
+author:
+- name: "Robert Gentleman"
+- name: "Vincent J. Carey"
+- name: "Wolfgang Huber"
+- name: "Florian Hahne"
+- name: "Emmanuel Taiwo"
+  affiliation: "Vignette translation from Sweave to R Markdown / HTML" 
+date: "`r format(Sys.time(), '%B %d, %Y')`" 
+vignette: > 
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{How to find genes whose expression profile is similar to 
+  that of specified genes} 
+  %\VignettePackage{genefilter} 
+  %\VignetteEncoding{UTF-8}
+output: 
+  BiocStyle::html_document: 
+    number_sections: yes 
+    toc: yes 
+    toc_depth: 4
+---
+
+# Introduction 
+
+In some cases you have certain genes of interest and you would like to find
+other genes that are close to the genes of interest. This can be done using the
+`genefinder` function.
+
+You need to specify either the index position of the genes you want (which row
+of the expression array the gene is in) or the name (consistent with the
+`featureNames` of the ExpressionSet).
+
+A vector of names can be specified and matches for all will be computed. The
+number of matches and the distance measure used can all be specified. The
+examples will be carried out using the artificial data set, `sample.ExpressionSet`.
+
+Two other options for `genefinder` are `scale` and `method`. The `scale` option
+controls the scaling of the rows (this is often desirable), while the `method`
+option controls the distance measure used between genes. The possible values and
+their meanings are listed at the end of this document.
+
+```{r, message = FALSE}
+library("Biobase")
+```
+
+```{r}
+library("genefilter")
+data(sample.ExpressionSet)
+igenes <- c(300,333,355,419) ##the interesting genes
+closeg <- genefinder(sample.ExpressionSet, igenes, 10, method="euc", scale="none")
+names(closeg)
+```
```

Comment:
Please combine into a single block as in the PDF.

Reply:
Done